django/otto/models.py

```diff
@@ -86,6 +86,10 @@ def roles(self):
     def display_total_cost(self):
```

Comment:
Looks like we no longer need this function anymore, since it's not used anywhere?